NEWS.md

```diff
@@ -192,24 +192,24 @@
     ```R
     set.seed(108)
     x = rnorm(1e6); n = 1e3
-    rollfun = function(x, n, FUN) {
-      nx = nx = length(x)
+    base_rollapply = function(x, n, FUN) {
+      nx = length(x)
       ans = rep(NA_real_, nx)
       for (i in n:nx) ans[i] = FUN(x[(i-n+1):i])
       ans
     }
-    system.time(rollfun(x, n, mean))
+    system.time(base_rollapply(x, n, mean))
     system.time(zoo::rollapplyr(x, n, function(x) mean(x), fill=NA))
     system.time(zoo::rollmeanr(x, n, fill=NA))
     system.time(frollapply(x, n, mean))
     system.time(frollmean(x, n))
 
     ### fun             mean     sum  median
-    # base rollfun     8.815   5.151  60.175
+    # base_rollapply   8.815   5.151  60.175
     # zoo::rollapply  34.373  27.837  88.552
-    # zoo::roll[fun]   0.215   0.185      NA
+    # zoo::roll[fun]   0.215   0.185      NA   ## median not fully supported
     # frollapply       5.404   1.419  56.475
-    # froll[fun]       0.003   0.002      NA
+    # froll[fun]       0.003   0.002      NA   ## median not yet supported
```

Comment:
@jangorecki Ahah: I had not guessed that was the reason for the NA (i.e. median not supported).  When I was reading quickly like most people,  I thought it was mean, sum and median of a set of timings,  a bit like microbenchmark output.  Now it makes sense!